code/stat159lambda/classification/partition_volumes.py

```diff
@@ -3,12 +3,14 @@
 from stat159lambda.config import REPO_HOME_PATH, NUM_OFFSET_VOLUMES, NUM_VOLUMES
 from stat159lambda.utils import scene_slicer as ss 
 
+TRAINING_PERCENT = .80
```

Comment:
This isnt a project wide constant. its just used in this file so i dont want it in config